R/commonMachineLearningClassification.R

```diff
@@ -229,6 +229,9 @@
     classificationTable$addRows(row)
 
   }
+
+  if(options[["saveModel"]])
+    saveRDS(classificationResult[["model"]], options[["file"]])
```

Comment:
I would create a custom class here, something like:

```r
if(options[["saveModel"]]) {
  # possibly add or remove things we do or don't need
  class(classificationResult) <- c("jaspClassification" "jaspMachineLearning")
  saveRDS(classificationResult[["model"]], options[["file"]])
}
```

I'd also add the function
```r
is.jaspMachineLearning <- function(x) inherits(x, "jaspMachineLearning")
```